### PR TITLE
doc: Adding help for core benchmark in jenkins

### PR DESCRIPTION
**Pull request by gareth-ellis**
First draft of document for running core benchmarks through jenkins.

@nodejs/benchmarking , please take a look, have a go at running the benchmarks (preferably start with a small subset!), and give feedback on this doc.

Likewise if you come across any functional issues, please comment in #127

### CODE REVIEW

File: docs/core_benchmarks.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# How to: Running core benchmarks on Node.js CI
+
+## Current function
+The current jenkins job aims at allowing the impact of a PR on a particular branch to be compared. For example how does PR XXXX impact performance on master?
+
+## How to configure the job
+To start with visit https://ci.nodejs.org/job/benchmark-node-micro-benchmarks/build?delay=0sec
+Here you have four values to enter:
+### BRANCH
+this is which branch or tag to compare to, defaults to master
```
Comment (mhdawson) [inline docs/core_benchmarks.md]: Capital at start of line here.

File: docs/core_benchmarks.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+### PULL_ID
+pull request to test
```
Comment (mhdawson) [inline docs/core_benchmarks.md]: Capital at start of line here.

File: docs/core_benchmarks.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+These two variables provide the two copies of Node.js to compare - we build BRANCH, then take a copy and apply the diff of PULL_ID, and build again. We now have two copies of Node to compare
+
+### CATEGORY
+This is a category of benchmarks in the node/benchmark folder from core
+### FILTER
+this reduces further the benchmarks in scope and is includes benchmark where FILTER exists in the filename.
```
Comment (mhdawson) [inline docs/core_benchmarks.md]: Capital at start of line here.

**Comment by mhdawson** [comment]
@nodejs/benchmarking any other comments ?

**Comment by gareth-ellis** [comment]
Comments addressed

**Comment by mhdawson** [comment]
Landed as 8eb28ef4dce0c77f61172f35c1e227332cabc709

**Pull request closed without merging**